docs/interfaces.md

```diff
@@ -45,6 +45,14 @@ apps interfering with one another.
 Usage: reserved
 Auto-Connect: yes
 
+### pulseaudio
+
+Can access the PulseAudio sound server. Allows for sound playback in games and
+media application.
+
+Usage: common
+Auto-Connect: yes
```

Comment:
@niemeyer - @morphis created a PoC pulseaudio plugin that examines the apparmor label of the connecting process and if it starts with 'snap.', disallow recording. Doing this in the immediate term would satisfy the security concerns until the proper mediation for recording is available.